### ISSUE TITLE
Implement web Push API for notifications

### ISSUE THREAD

**Issue by raintonr** [ioBroker/ioBroker.lovelace#420]
The notifications bell is useful, but could be even more so if Web Push API was used so one could be alerted on their mobile, for example, even when the interface was closed.

**Comment by Garfonso** [comment]
Shouldn't that be a request for the frontend? The frontend is not developed in this project, but just adjusted for the use with ioBroker.

With that being said, I guess HomeAssistant has its own app for that which supports notification. Sadly I did not yet manage to add support for the HomeAssistant App. We add something about the messages to the frontend, already. If you know how, maybe you could have a look here and do a PR: https://github.com/GermanBluefox/home-assistant-polymer

On the other hand, there are ample adapters that can notify a mobile device, like example the telegram adapter.

**Comment by raintonr** [comment]
Yeah, I get that and in fact already use Telegram adapter. My point is that if a mechanism were available for notifications that avoided the need for such third party services many people would probably prefer that.

**Comment by Garfonso** [comment]
I once again checked this request. This will still need third party services, in this case HomeAssistant. They probably won't allow us on them. 
Of course a similar feature could be implementend on our own, but this is out of scope for this adapter, sadly.
